owncloudApp/src/main/java/com/owncloud/android/presentation/ui/settings/fragments/SettingsFragment.kt

```diff
@@ -204,6 +221,112 @@ class SettingsFragment : PreferenceFragmentCompat() {
+                val syncUrl = settingsViewModel.getSyncUrl()
+                goToUrl(syncUrl)
+                true
+            }
+        } else {
+            prefMoreCategory?.removePreference(prefSync)
+        }
+
+        // Recommend
+        if (settingsViewModel.isRecommendEnabled()) {
+            prefRecommend?.setOnPreferenceClickListener {
+                val intent = Intent(Intent.ACTION_SENDTO)
+                intent.data = Uri.parse(getString(R.string.mail_recommend))
+                intent.addFlags(Intent.FLAG_ACTIVITY_NEW_TASK)
+
+                val appName = getString(R.string.app_name)
+                val downloadUrl = getString(R.string.url_app_download)
+
+                val recommendSubject = String.format(getString(R.string.recommend_subject), appName)
+                val recommendText = String.format(getString(R.string.recommend_text), appName, downloadUrl)
+
+                intent.putExtra(Intent.EXTRA_SUBJECT, recommendSubject)
+                intent.putExtra(Intent.EXTRA_TEXT, recommendText)
+                startActivity(intent)
+                true
+            }
+        } else {
+            prefMoreCategory?.removePreference(prefRecommend)
+        }
+
+        // Feedback
+        if (settingsViewModel.isFeedbackEnabled()) {
+            prefFeedback?.setOnPreferenceClickListener {
+                val feedbackMail = getString(R.string.mail_feedback)
+                val feedback = "Android v" + BuildConfig.VERSION_NAME + " - " + getString(R.string.prefs_feedback)
+                val intent = Intent(Intent.ACTION_SENDTO)
+                intent.putExtra(Intent.EXTRA_SUBJECT, feedback)
+
+                intent.data = Uri.parse(feedbackMail)
+                intent.addFlags(Intent.FLAG_ACTIVITY_NEW_TASK)
+                startActivity(intent)
+                true
+            }
+        } else {
+            prefMoreCategory?.removePreference(prefFeedback)
+        }
+
+        // Privacy policy
+        if (settingsViewModel.isPrivacyPolicyEnabled()) {
+            prefPrivacyPolicy?.setOnPreferenceClickListener {
+                val intent = Intent(context, PrivacyPolicyActivity::class.java)
+                startActivity(intent)
+                true
+            }
+        } else {
+            prefMoreCategory?.removePreference(prefPrivacyPolicy)
+        }
+
+        // Imprint
+        if (settingsViewModel.isImprintEnabled()) {
+            prefImprint?.setOnPreferenceClickListener {
+                val imprintUrl = settingsViewModel.getImprintUrl()
+                goToUrl(imprintUrl)
+                true
+            }
+        } else {
+            prefMoreCategory?.removePreference(prefImprint)
+        }
+
+        // About app
+        prefAboutApp?.let {
+            it.title = String.format(getString(R.string.about_android), getString(R.string.app_name))
+            val appVersion = BuildConfig.VERSION_NAME + " " + BuildConfig.BUILD_TYPE + " " + BuildConfig.COMMIT_SHA1
```

Comment:
Just a suggestion, nothing relevant. But I prefer this way to concatenate strings in Kotlin. Up to you 👍           
```suggestion
            val appVersion = "${BuildConfig.VERSION_NAME} ${BuildConfig.BUILD_TYPE} ${BuildConfig.COMMIT_SHA1}"

```